tracer/src/Datadog.Trace.Tools.Runner/Checks/Resources.cs

```diff
@@ -13,11 +15,17 @@ internal static class Resources
         public const string NetFrameworkRuntime = "Target process is running with .NET Framework";
         public const string NetCoreRuntime = "Target process is running with .NET Core";
         public const string RuntimeDetectionFailed = "Failed to detect target process runtime, assuming .NET Framework";
+        public const string BothRuntimesDetected = "The target process is running .NET Framework and .NET Core simultaneously. Checks will be performed assuming a .NET Framework runtime.";
```

Comment:
Do we actually _support_ this mode? If not, is it worth mentioning that?

Reply:
This is indicated later in the `IisMixedRuntimes` message